lib/ecto/changeset.ex

```diff
@@ -573,7 +573,7 @@ defmodule Ecto.Changeset do
   This function should be used when working with the entire association at
   once (and not a single element of a many-style association) and using data
   external to the application.
-  
+
```

Comment:
Trailing spaces are removed by the editor, please tell if you don't want this change.